### DISCUSSION TITLE
CompreFace integration

### DISCUSSION THREAD

**Discussion by parkroyk** [roflcoopter/viseron#598] [Q&A]
With CompreFace running with Viseron, are we able to draw object outlines with the person's names being recognized?

It would be also tremendously useful to be able to store/search recordings with the face (and object) tags. Is that supported currently?

**Comment by roflcoopter** [comment]
I am working on the foundation of this feature in this issue #455 

It will allow users to configure retention periods for snapshots. Currently these are not saved anywhere sadly

**Comment by hessijames79** [comment]
Hi! I would also be interested to be able to quickly check the quality of face recognition results and to be able to fine tune confidences. Is there currently any way to get e. g. a folder with all detected faces tagged/grouped by the name? A solution without Viseron would also work.

Patrick

**Reply by roflcoopter** [reply]
Viseron sends coordinates and other data over MQTT when a face is detected, but not together with the image.
The detections are not saved to disk either sadly.

I have to the basics working now for #455 so it shouldnt be too long until i can implement this
